chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
+for managing the process of collaboration on any technical product.
+
+For most research projects, completing a written piece is not the end of the task.
+In almost all cases, you will be required to release a replication package,
+which contains the code and materials needed to create the results.
+These represent an intellectual contribution in their own right,
+because they enable others to learn from your process
+and better understand the results you have obtained.
+Holding code and data to the same standards as written work
+is a new discipline for many researchers,
+and here we provide some basic guidelines and responsibilities for both
+that will help you prepare a functioning and informative replication package.
+In all cases, we note that technology is rapidly evolving
+and that the specific tools noted here may not remain cutting-edge,
+but the core principles involved in publication and transparency will endure.
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Collaborating on academic writing}
+\section{Collaborating on technical writing}
+
+It is increasingly rare that a single author will prepare an entire manuscript alone.
+More often than not, documents will pass back and forth between several writers
+before they are ready for publication,
+so it is essential to use technology and workflows that avoid conflicts.
+Just as with the preparation of analytical outputs,
+this means adopting tools and practices that enable tasks
+such as version control and simultaneous contribution.
+Furthermore, it means preparing documents that are \textbf{dynamic} --
+meaning that updates to the analytical outputs that constitute them
+can be passed on to the final output with a single process,
+rather than copy-and-pasted or otherwise handled individually.
+Thinking of the writing process in this way
+is intended to improve organization and reduce error,
+such that there is no risk of materials being compiled
+with out-of-date results, or of completed work being lost or redundant.
+
+\subsection{Dynamic documents}
+
+Dynamic documents are a broad class of tools that enable such a workflow.
+The term ``dynamic'' can refer to any document-creation technology
+that allows the creation of explicit references to raw output files.
+This means that, whenever outputs are updated,
+the next iteration of the document will automatically include
+all changes made to all outputs without any additional intervention from the writer.
+This means that updates will never be accidentally excluded,
+and it further means that updating results will not become more difficult
+as the number of inputs grows,
+because they are all managed by a single integrated process.
+
+You will note that this is not possible in tools like Microsoft Office.
+In Word, for example, you have to copy and paste each object individually
+whenever tables, graphs or other inputs have to be updated.
+This means that both the features above are not available:
```

Comment:
"_both the features above_" - after reading several times I am still not sure which two they are

Reply:
859e01a1df175ff58612255636d77c96422a5836